src/datasource_application.ts

```diff
@@ -147,7 +151,6 @@ export default class InstanaApplicationDataSource extends AbstractDatasource {
       metric['granularity'] = target.timeInterval.value;
     }
 
-
     const group = {
       groupbyTag: target.group.key
     };
```

Comment:
Comment for line: 158
Why only one field as secondary option ?
It should be all fields with "type":"KEY_VALUE_PAIR" from server.

Wishlist would be at least for the most commonly used ones:
- docker.label
- kubernetes.pod.label
- call.http.params
- kubernetes.label
- aws.ec2.tag